Archive/Arithcc.lean

```diff
@@ -309,17 +309,17 @@ theorem compiler_correctness :
         (∀ x, loc x map < t) → outcome (compile map e t) η ≃[t] { η with ac := value e ξ } := by
   intro map e ξ η t hmap ht
   revert η t
-  induction e <;> intro η t hmap ht
+  induction e with intro η t hmap ht
```

Comment:
Can we use `induction .. generalizing` instead of `revert; induction` here? Or you prefer to leave it to another PR?

Reply:
sure, we can remove all the `intro` as well.